interfaces/builtin/docker_support.go

```diff
@@ -175,8 +175,22 @@ ptrace (read, trace) peer=cri-containerd.apparmor.d,
 # needed by runc for mitigation of CVE-2019-5736
 # For details see https://bugs.launchpad.net/apparmor/+bug/1820344
 / ix,
+/bin/busybox ixr,
 /bin/runc ixr,
 /pause ixr,
+
+# When kubernetes drives containerd, containerd needs access to CNI services,
+# like flanneld's subnet.env for DNS. This would ideally be snap-specific (it
+# could if the control plane was a snap), but in deployments where the control
+# plane is not a snap, it will tell flannel to use this path.
+/run/flannel/{,**} rk,
```

Comment:
I understand why we needed this in kubernetes-support for microk8s, but why do we also need it in docker-support?

Reply:
The control plane tells containerd where to find information when flanneld is in use and containerd needs to be able to read it. In this case with flannel, it is to help with DNS.

Reply:
ah I see, that makes sense I didn't read closely enough